servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -184,11 +183,14 @@ public int consecutive5xx() {
     }
 
     /**
-     * The interval on which to run failure percentage and success rate failure detectors.
+     * The interval on which to run failure detectors.
+     * Failure percentage and success rate outlier detectors perform periodic scans to detect outliers. Active
+     * revival mechanisms such as the layer-4 connectivity detector also use this interval to perform their periodic
+     * health check to see if a host can be considered revived.
      * @return the interval on which to run failure percentage and success rate failure detectors.
      */
-    public Duration interval() {
-        return interval;
+    public Duration scanInterval() {
```

Comment:
WDYT about `failureDetectorInterval()` and `failureDetectorIntervalJitter()`? It's nice when name matches with javadoc wording.